ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -2,33 +2,36 @@
 import os
 import uuid
 
+from dcos_installer import backend
+
 
 class InstallerAnalytics():
-    def __init__(self, config):
+    def __init__(self):
         self.uuid = self.new_uuid()
```

Comment:
We do this exactly once, and the new_uuid() code is really short / simple `uuid.uuid4()`. Lets just put that inline here. Adding the abstraction of calling a function isn't adding a lot of value.